### ISSUE TITLE
Mobilenet transfer learning has a higher final test accuracy than Inception V3 and Inception ResNet v2

### ISSUE THREAD

**Issue by marcreyesph** [tensorflow/tensorflow#24715]
I tried and did some experimenting on running the [retrain.py](https://github.com/tensorflow/hub/blob/master/examples/image_retraining/retrain.py) script for transfer learning. I did the transfer learning of images with 512px x 384px, of the same learning rate (used 0.01) and training steps (used 10000) using [Inception V3](https://tfhub.dev/google/imagenet/inception_v3/feature_vector/1), [Mobilenet_v2_140_224](https://tfhub.dev/google/imagenet/mobilenet_v2_140_224/feature_vector/2), and [Inception ResNet V2](https://tfhub.dev/google/imagenet/inception_resnet_v2/feature_vector/1). I found out that while and after retraining, Mobilenet_v2_140_224 has a higher accuracy compared to Inception V2 and Inception ResNet V3. Below are the final test accuracy of the retraining:

- Mobilenet_v2_140_224: **0.97790337 (97.9%)**
- Inception V3: **0.93268245 (93.3%)**
- Inception ResNet V2: **0.9116136 (91.2%)**

What could possibly be the cause of this? Thanks a lot.

**Comment by freedomtan** [comment]
Kinda overfitting probably. Mobilenet V2 has much fewer parameters than the other two, so this looks reasonable to me.

**Comment by jvishnuvardhan** [comment]
Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. Github is mainly for addressing bugs in installation and performance. Thanks!

**Comment by jvishnuvardhan** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by fengwuxuan** [comment]
@marcreyesph I have same question about myself dastaset, what's the reason?
